src/ejabberd_admin.erl

```diff
@@ -169,6 +170,13 @@ commands() ->
                         result = {res, restuple}}
     ].
 
+%%%
+%%% Commands
+%%%
+-spec status() -> {ok, {boolean(), iolist()}} | {mongooseim_not_running, iolist()}.
+status() ->
+    {ok, {Status, Message, _}} = mongoose_server_api:status(),
+    {ok, {Status, Message}}.
```

Comment:
Could you explain why adding this function here was needed? Because I can't figure it out.

Reply:
The changes resulted in mongoose_server_api:status() returning the tuple of three(Status, Message, Version) instead of a tuple of two(Status, Message). The additional element in a tuple is used by mongoose_graphql_server_admin_query, to return an additional field in a graphql response. However, we don't want to change the ejabberd_admin status command(which by the way I think is an unused and just dead code apparently). Before the change, it was just calling mongoose_server_api:status, but when I changed it, I needed to replace it with a function, that would call mongoose_server_api:status and remove one additional element from the tuple.

Reply:
Ok, I see. Thanks for explaining.